homeassistant/components/zwave_js/api.py

```diff
@@ -67,7 +67,9 @@
 PROPERTY = "property"
 PROPERTY_KEY = "property_key"
 VALUE = "value"
-SECURE = "secure"
+INCLUSION_STRATEGY = "inclusion_strategy"
+INCLUSION_GRANT = "inclusion_strategy"
```

Comment:
Typo?

Reply:
more like moving too quickly through copy and pastes 😅